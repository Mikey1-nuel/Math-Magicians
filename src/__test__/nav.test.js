import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Nav from '../components/nav';

describe('Nav component', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Nav />
        </BrowserRouter>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});