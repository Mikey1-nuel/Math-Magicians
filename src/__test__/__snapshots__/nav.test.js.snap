// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component renders correctly 1`] = `
<nav
  className="nav"
>
  <h1>
    Math Magician
  </h1>
  <ul
    className="menu"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      <li
        className="navlink"
        title="home"
      >
        Home
      </li>
    </a>
    <a
      href="/calculator"
      onClick={[Function]}
    >
      <li
        className="navlink"
        title="calculator"
      >
        Calculator
      </li>
    </a>
    <a
      href="/quotes"
      onClick={[Function]}
    >
      <li
        className="navlink"
        title="quotes"
      >
        Quotes
      </li>
    </a>
  </ul>
</nav>
`;
